ArduinoCode/Pedal_Potentiometers/Pedal_Potentiometers.ino

```diff
@@ -99,20 +107,21 @@ void readDashboard() {
       CAN.readMsgBuf(&len, buf); //enters message into program
       canId = CAN.getCanId(); //gets sender ID
      if (canID == 0x01) {  //determine whether Dashboard is on or off
-      if (buf[0] == 0) {
-       // code for when dashboad is off
+      if (buf[0] == 0) {  // motor is off
+       invertorOn = false;
+       MotorOff();
```

Comment:
You can call MotorOff() outside of this if statement as anytime a message is received from the id 0x01 the motor should be turned off (even when message is 1)